master/internal/trial.go

```diff
@@ -1030,8 +1030,12 @@ func classifyStatus(state terminatedContainerWithState) aproto.ContainerStopped
 }
 
 func (t *trial) reset() error {
-	step := t.sequencer.RollBackSequencer()
-	if err := t.db.RollBackTrial(t.id, step); err != nil {
+	totalBatches, err := t.sequencer.RollBackSequencer()
+	if err != nil {
+		return errors.Wrap(err, "failed to get workload from sequencer in reset")
```

Comment:
oh I see this now.. may as well wrap the other though (`RollBackSequencer`) and have this not say what the function it called was doing.